modules/health_quest/config/mock/appointment.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
How did you generate this file? I'm not familiar with the schema.

Reply:
Great question! The file was composed after speaking with MAP and FE developers. The structure of this appointment was molded after `/vets-api-mockdata/ihub/appointments/default.yml` from the vets-api-mockdata.

Reply:
Another source that we used to build the appointment file: `vets-api/spec/support/vcr_cassettes/vaos/appointment_requests/get_requests.yml`